storage/inkless/src/main/java/io/aiven/inkless/common/metrics/ThreadPoolMonitor.java

```diff
@@ -60,19 +62,23 @@ public ThreadPoolMonitor(final String groupName, final ExecutorService pool) {
 
         // ThreadPoolExecutor monitoring
         this.activeThreadsSensor = new SensorProvider(metrics, ACTIVE_THREADS)
-            .with(metricsRegistry.activeThreadsTotalMetricName, new MeasurableValue(() -> (long) threadPoolExecutor.getActiveCount()))
+            .with(metricsRegistry.activeThreadsTotalMetricName, new MeasurableValue<>(() -> (long) threadPoolExecutor.getActiveCount()))
             .get();
 
         this.poolSizeSensor = new SensorProvider(metrics, POOL_SIZE)
-            .with(metricsRegistry.poolSizeTotalMetricName, new MeasurableValue(() -> (long) threadPoolExecutor.getPoolSize()))
+            .with(metricsRegistry.poolSizeTotalMetricName, new MeasurableValue<>(() -> (long) threadPoolExecutor.getPoolSize()))
             .get();
 
         this.parallelismSensor = new SensorProvider(metrics, PARALLELISM)
-            .with(metricsRegistry.parallelismTotalMetricName, new MeasurableValue(() -> (long) threadPoolExecutor.getCorePoolSize()))
+            .with(metricsRegistry.parallelismTotalMetricName, new MeasurableValue<>(() -> (long) threadPoolExecutor.getCorePoolSize()))
             .get();
 
         this.queuedTaskCountSensor = new SensorProvider(metrics, QUEUED_TASK_COUNT)
-            .with(metricsRegistry.queuedTaskCountTotalMetricName, new MeasurableValue(() -> (long) threadPoolExecutor.getQueue().size()))
+            .with(metricsRegistry.queuedTaskCountTotalMetricName, new MeasurableValue<>(() -> (long) threadPoolExecutor.getQueue().size()))
+            .get();
+
+        this.avgIdlePercentSensor = new SensorProvider(metrics, AVG_IDLE_PERCENT)
+            .with(metricsRegistry.avgIdlePercentMetricName,  new MeasurableValue<>(() -> 1 - (double) threadPoolExecutor.getActiveCount() / (double) threadPoolExecutor.getCorePoolSize()))
```

Comment:
WDYT of this change?
```suggestion
            .with(metricsRegistry.avgIdlePercentMetricName,  new MeasurableValue<>(() -> 1 - (double) threadPoolExecutor.getActiveCount() / (double) threadPoolExecutor.getPoolSize()))
```
With the current formula, if the number of active threads goes above the core pool size you will get a negative average.